.devcontainer/development/Dockerfile

```
@@ -65,6 +65,9 @@ ENV PATH="${PATH}:/root/.local/bin"
 # Install Gemini CLI.
 RUN npm install -g @google/gemini-cli
 
+# Install GitHub Copilot CLI (standalone coding agent with model selection)
+RUN npm install -g @github/copilot
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)